docs/pages/guides/mesheryctl.md

```diff
@@ -13,6 +13,7 @@ Installation, troubleshooting and debugging of Meshery and its adapaters.
 |:----------|:-----:|:--------------------------|:--------------------------|
 |cleanup    |       |Pulls current meshery.yaml from meshery repo. *Warning: Any local changes will be overwritten.* | `mesheryctl cleanup` |
 |log        |       |Starts tailing Meshery debug logs.              | `mesheryctl log` |
+|perf       |       |Performance testing and benchmarking.              | `mesheryctl perf ` |
```

Comment:
Let's keep `perf` documented under the **Performance Management** group of commands and avoid combining it will Meshery Lifecycle Management commands. Please revert this update, if you would.